### PR TITLE
Importing a calendar/event doesn't send a mail now

### PR DESCRIPTION
**Pull request by Julian1998**
Fixing https://github.com/owncloud/core/issues/23159#issuecomment-363975475

### CODE REVIEW

File: js/app/service/veventService.js [hunk 1]
```diff
@@ -160,13 +160,21 @@ app.service('VEventService', function(DavClient, StringUtility, XMLUtility, VEve
...
+		var headers = {
 			'Content-Type': 'text/calendar; charset=utf-8',
 			'requesttoken': OC.requestToken
 		};
+
+		if(isImport) {
+			headers['OC-CalDav-Import'] = true;
+		}
+
+		console.log(headers);
```
Comment (DeepDiver1975) [inline js/app/service/veventService.js]: please remove logging - THX

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

File: js/app/service/veventService.js (hunk 1 above)
Comment (Julian1998) [reply js/app/service/veventService.js]: Oh lol haha.... It was too late... my bad

**Comment by Julian1998** [comment]
git rebase

**Comment by DeepDiver1975** [comment]
squash and rebase please

**Review by DeepDiver1975 — APPROVED** [review]

**Pull request merged**